roles/calico/templates/calico.service.j2

```diff
@@ -10,8 +10,12 @@ ExecStart=/usr/bin/docker run --net=host --privileged \
  --name=calico-node \
  -e WAIT_FOR_DATASTORE=true \
  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
- -e CALICO_IPV4POOL_IPIP=always \
- -e FELIX_IPV6SUPPORT=false \
+ -e CALICO_IPV4POOL_IPIP={{ calico_ipv4pool_ipip }} \
+ -e CALICO_IPV4POOL_CIDR={{ calico_ipv4pool_cidr }} \
+ {% if felix_ipv6support is not defined or true %}
+ -e CALICO_IPV6POOL_CIDR={{ calico_ipv6pool_cidr }} \
+ {% endif %}
+ -e FELIX_IPV6SUPPORT={{ felix_ipv6support }} \
```

Comment:
This shouldn't even be an option. Neither Kubernetes nor OpenShift support IPv6 for pods.